src/components/My Profile page/BasicInfo.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useRef } from 'react';
+import {
+    Box,
+    Button,
+    FormControl,
+    FormLabel,
+    Heading,
+    Input,
+    Stack,
+    InputGroup,
+} from '@chakra-ui/react';
+
+const BasicInfo = () => {
+    const [name, setName] = useState('Michael Jones');
+
+    const fileChooser = useRef(null);
+
+    const profileInfo = (e) => {
```

Comment:
Functions should be named as a verb + noun instead of noun, eg here it may be `submitProfileInfo`

```suggestion
    const profileInfo = (e) => {
```